jcl/src/java.base/share/classes/java/lang/String.java

```diff
@@ -8360,31 +8360,44 @@ public String stripIndent() {
 		if (isEmpty()) {
 			return this;
 		}
+
+		boolean trailingNewLine = false;
+		char lastChar = charAt(length() - 1);
+
+		if (('\n' == lastChar) || ('\r' == lastChar)) {
+			trailingNewLine = true;
+		}
```

Comment:
I'd prefer to see this folded into the initializer:
```
char lastChar = charAt(length() - 1);
boolean trailingNewLine = ('\n' == lastChar) || ('\r' == lastChar);
```

Reply:
updated